.github/workflows/go-e2e.yaml

```diff
@@ -9,6 +9,8 @@ on:
     types: [opened, reopened, ready_for_review, synchronize] # added `ready_for_review` since draft is skipped
     paths:
       - microservices-connector/**
+      - manifests/ChatQnA/**
+      - manifests/DocSum/xeon/docsum*
```

Comment:
shall we add the other two examples(codegen and codetrans) as well?

Reply:
I think we've already added the test cases of codegen and codetrans. Yet GMC doesn't have any dependencies to manifest/CodeGen and manifest/CodeTrans. GMC only uses manifest/ChatQnA and manifests/DocSum/*/docsum_llm.yaml. So I just include these two folders in the workflow.